store/store.go

```diff
@@ -467,6 +467,9 @@ type requestOptions struct {
 	Data        []byte
 }
 
+// Store method for doing requests to store server
+var doStoreRequest = (*Store).doRequest
```

Comment:
this is a bit odd, I would prefer a middle ground where we still check some property of requests to know doRequest was called